workers/data_refinery_workers/processors/utils.py

```diff
@@ -197,7 +197,8 @@ def start_job(job_context: Dict):
         raise Exception("processors.start_job called on job %s that has already been started!" % str(job.id))
 
     original_file = job.original_files.first()
-    if original_file and not original_file.needs_processing(job_context["job_id"]):
+    if not job.pipeline_applied == ProcessorPipeline.TXIMPORT and original_file\
```

Comment:
I'm fairly certain this needs to be `ProcessorPipeline.TXIMPORT.value` because job.pipeline_applied is a string and ProcessorPipeline.TXIMPORT is an Enum.

Reply:
Good catch